### ISSUE TITLE
Next stabilisation 

### ISSUE THREAD

**Issue by CorrosiveTruths** [Zygo/bees#122]
Hello, unfortunately, version 0.6.1 doesn't work for me, the uint64_t
max is max_transid in beescrawl so the process never finishes. The latest git version works fine.

Will there be another stable release soonish? Or would you recommend the git verion?

**Comment by kakra** [comment]
A fix for this should be in v0.6.1 already. See c4267945425063bf053c2b309a02ebc088177e62

**Comment by CorrosiveTruths** [comment]
> A fix for this should be in v0.6.1 already. See [c426794](https://github.com/Zygo/bees/commit/c4267945425063bf053c2b309a02ebc088177e62)

That does look like related and thank you for addressing this, yet I go back to 0.6.1 and run bees after snapshotting some subvols and I get lines like these:

root 24263 objectid 4138656 offset 1935 min_transid 117213 max_transid 18446744073709551615 started 1572385072 start_ts 2019-10-29-21-37-52
root 24264 objectid 0 offset 0 min_transid 117213 max_transid 18446744073709551615 started 1572385072 start_ts 2019-10-29-21-37-52
root 24265 objectid 2658316 offset 1936 min_transid 117213 max_transid 18446744073709551615 started 1572385072 start_ts 2019-10-29-21-37-52
root 24266 objectid 1621246 offset 1403494 min_transid 117213 max_transid 18446744073709551615 started 1572385073 start_ts 2019-10-29-21-37-53
root 24267 objectid 0 offset 0 min_transid 117213 max_transid 18446744073709551615 started 1572385073 start_ts 2019-10-29-21-37-53
root 24268 objectid 629910 offset 1 min_transid 117213 max_transid 18446744073709551615 started 1572385073 start_ts 2019-10-29-21-37-53

Instead of these ones from the last time I ran git master:

root 24263 objectid 0 offset 0 min_transid 117204 max_transid 117213 started 1572348391 start_ts 2019-10-29-11-26-31
root 24264 objectid 0 offset 0 min_transid 117204 max_transid 117213 started 1572348391 start_ts 2019-10-29-11-26-31
root 24265 objectid 0 offset 0 min_transid 117204 max_transid 117213 started 1572348391 start_ts 2019-10-29-11-26-31
root 24266 objectid 0 offset 0 min_transid 117204 max_transid 117213 started 1572348391 start_ts 2019-10-29-11-26-31
root 24267 objectid 0 offset 0 min_transid 117204 max_transid 117213 started 1572348391 start_ts 2019-10-29-11-26-31
root 24268 objectid 0 offset 0 min_transid 117204 max_transid 117213 started 1572348391 start_ts 2019-10-29-11-26-31

These are both after stopping the service after crawl_writeback comes back clean. 0.6.1 redoes work when I restart it and git master doesn't (and restarts without systemd killing beesd to boot).

Looking at the commits since 0.6.1: https://github.com/Zygo/bees/compare/v0.6.1...master I can see some which look like they address my issue, like https://github.com/Zygo/bees/commit/866a35c7fb8c06ce6a6f130af979a61c21b2ee21 looks super pertinent?

Anyway, tl;dr, bees from git master works better than 0.6.1 for me, but I can just use git master until / if a next version is released, so no biggie.

**Comment by CorrosiveTruths** [comment]
Sorry just noticed you edited your post, let me check out #130.

**Comment by CorrosiveTruths** [comment]
Not seeing anything that looks like it helps me in #130, I'm afraid?

**Comment by kakra** [comment]
You shouldn't compare v0.6.1 to master this way because it may show you commits that have been cherry-picked into the v0.6 branch since the branch point. Especially, 866a35c is contained in that branch as a cherry-pick.

Use `git cherry -v master v0.6.1` to find `ce0c1ab6298e039e518621500367d3586c86a1a0` being part of v0.6.1 which is a cherry-pick of the "super pertinent" commit.

**Comment by kakra** [comment]
> Not seeing anything that looks like it helps me in #130, I'm afraid?

Actually, a5b9919d26 may be the commit you are missing from v0.6.1

**Comment by CorrosiveTruths** [comment]
> You shouldn't compare v0.6.1 to master this way because it may show you commits that have been cherry-picked into the v0.6 branch since the branch point [1]. Especially, [866a35c](https://github.com/Zygo/bees/commit/866a35c7fb8c06ce6a6f130af979a61c21b2ee21) is contained in that branch as a cherry-pick.
> 
> Use `git cherry -v master v0.6.1` to find [ce0c1ab](https://github.com/Zygo/bees/commit/ce0c1ab6298e039e518621500367d3586c86a1a0) being part of v0.6.1 which is a cherry-pick of the "super pertinent" commit.
> 
> [1]: The history is not linear, they branched at v0.6 and the commits you mentioned went into master after the branch point and have been cherry-picked. A direct compare in Github will list them as "changed between" which is somewhat true and somewhat false. Please, use `cherry` for such queries. ;-)

Oh yes, I see what you mean, I didn't know that, thank you for explaining and sorry if I confused things.

> Actually, https://github.com/Zygo/bees/commit/a5b9919d260888369375b7897745fb780dd2e879 may be the commit you are missing from v0.6.1

Yes, that seems to have done it, thank you - works with that pacth. On the other hand if I apply all of #130 I get a segfault?

**Comment by kakra** [comment]
> Oh yes, I see what you mean, I didn't know that, thank you for explaining and sorry if I confused things.

NP, happy to clear things up. You're welcome.
